src/dhtproto/client/UsageExamples.d

```diff
@@ -26,6 +26,21 @@
 
 module dhtproto.client.UsageExamples;
 
+
+/// Mixin helper for the SmartUnion's final switch handlers. It inserts
+/// case none: assert(false); in D2 and default: assert(false); in D1
+/// TODO: This should be replaced with the
+/// `ocean.core.SmartEnum.handleInvalidCases`, available in ocean v3.5.x and
+/// latter (not yet released).
+istring handleInvalidCases ()
```

Comment:
Does this need to be a function here or could it be a string?

Reply:
You're right, this is already addressed in the ocean. Updated.